groupsforsections.py

```diff
@@ -208,32 +208,44 @@ def main():
     logging.info('Script Started')
     logging.debug('args: ' + str(sys.argv))
 
-    if len(sys.argv) is not 2:
-        logging.error("The command line argument (path to properties file) is not provided")
+    if len(sys.argv) is not 3:
+        logging.error("Some of the command line arguments (path to properties files) are not provided")
```

Comment:
Maybe here you could print out the format in which the command is to be executed: python script.py ./path1 ./path2 ./path3